mandatory/4-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(number) {
+  return number + number*20/100;
```

Comment:
This can lead to a misleading result (or someone else may read this code differently). So it is always advisable to use parantheses. e.g.
```
number + (number * 20/100)
```